dirsrvtests/tests/suites/config/compact_test.py

```diff
@@ -72,6 +74,62 @@ def test_compaction_interval_and_time(topo):
     inst.deleteErrorLogs(restart=False)
 
 
+@pytest.mark.ds4778
+@pytest.mark.bz1748441
+@pytest.mark.skipif(ds_is_older("1.4.3.23"), reason="Not implemented")
+def test_no_compaction(topo):
+    """Test there is no compaction when nsslapd-db-compactdb-interval is set to 0
+
+    :id: 80fdb0e3-a70c-42ad-9841-eebb74287b19
+    :setup: Supplier Instance
+    :steps:
+        1. Configure nsslapd-db-compactdb-interval to 0
+        2. Check there is no compaction
+    :expectedresults:
+        1. Success
+        2. Success
+    """
+
+    inst = topo.ms["supplier1"]
+    config = DatabaseConfig(inst)
+    config.set([('nsslapd-db-compactdb-interval', '0'), ('nsslapd-db-compactdb-time', '00:01')])
+    inst.deleteErrorLogs()
+
+    time.sleep(6)
+    assert not inst.searchErrorsLog("Compacting databases")
+    inst.deleteErrorLogs(restart=False)
+
+
+@pytest.mark.ds4778
+@pytest.mark.bz1748441
+@pytest.mark.skipif(ds_is_older("1.4.3.23"), reason="Not implemented")
+def test_compaction_interval_invalid(topo):
+    """Test that invalid value is rejected for nsslapd-db-compactdb-interval
+
+    :id: 408ee3ee-727c-4565-8b08-2e07d0c6f7d7
+    :setup: Supplier Instance
+    :steps:
+        1. Set nsslapd-db-compactdb-interval to 2147483650
+        2. Check exception message contains invalid value and no compaction occurred
+    :expectedresults:
+        1. Exception is raised
+        2. Success
+    """
+
+    inst = topo.ms["supplier1"]
+    msg = 'value 2147483650 for attr nsslapd-db-compactdb-interval is greater than the maximum 2147483647'
+    config = DatabaseConfig(inst)
+
+    try:
+        config.set([('nsslapd-db-compactdb-interval', '2147483650'), ('nsslapd-db-compactdb-time', '00:01')])
+    except ldap.UNWILLING_TO_PERFORM as e:
+        log.info('Got expected error: {}'.format(str(e)))
+        assert msg in str(e)
+        time.sleep(6)
```

Comment:
Can you reduce this sleep time?  Maybe 3 seconds?  These "sleeps" add up when running the entire test suite.

Reply:
I reduced the sleep time, please review. Thanks!